synapse/cores/common.py

```diff
@@ -274,7 +281,8 @@ def _initCoreStormOpers(self):
     def _initCortexConfSetPre(self):
         # Setup handlers for confOpt changes which are not racy
         self.onConfOptSet('caching', self._onSetCaching)
-        self.onConfOptSet('axon:conf', self._onSetAxonConf)
+        self.onConfOptSet('cellpool:conf', self._onSetCellPoolConf)
```

Comment:
I plumbed these in separately to avoid needing to add more config options for other cells later

Reply:
Will need to move these since axon:name depends on the cellpool being configured